core/src/main/java/io/seata/core/store/db/DataSourceGenerator.java

```diff
@@ -25,10 +25,9 @@
 public interface DataSourceGenerator {
 
     /**
-     * create DataSource from config
-     *
+     * get the data source
      * @return data source
      */
-    DataSource generateDataSource();
+    DataSource get();
```

Comment:
retain generateDataSource() ?

Reply:
+1

Reply:
but generate() already in the AbstractDataSourceGenerator.generate(), this method just only get the data source.